gnome-extra/gnome-commander/gnome-commander-1.10.1.ebuild

```diff
@@ -1,18 +1,18 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
```

Comment:
any chance for an EAPI-7 bump here?

Reply:
Thanks for pointing that out. But let me postpone the EAPI-bump to the next version-bump of gnome-commander.